humanguide/design-concepts/design-philosophy.rst

```diff
@@ -0,0 +1,55 @@
+=================
+Design philosophy
+=================
+
+The Ubuntu interface has been designed according to a philosophy called Suru.
+
+.. figure:: /_static/images/humanguide/750w_DesignPhilosophy_MainImage.png
+
+
+Suru meaning
+------------
+
+Here at Ubuntu we believe that everyone should have access to free, reliable, and trusted software that can be shared and developed by anyone. This has paved the way to allow the community to grow and prosper with freedom trust and collaboration. This integral belief has been translates into our design philosophy called Suru.

```

Comment:
We're not Canonical. It's not prudent to say "Here at Ubuntu"

Reply:
has been translates ... grammar

Reply:
> 
> 
> has been translates ... grammar

yep..original text contains errors.